.github/workflows/cln_proxy_integration_test.yml

```diff
@@ -99,6 +99,10 @@ jobs:
         with:
           source: 'stack/relay/NODES.json'
           target: 'relay/src/tests/configs/nodes.json'
+      - name: Sleep for 1 minutes
+        uses: jakejarvis/wait-action@master
+        with:
+          time: '1m'
```

Comment:
why are we sleeping here can't we add another sleep in the check for NODES.json step?

Reply:
Bot setup only works after relay_setup has exited, so I am waiting 1 minute after relay_setup has exited for both bot containers to come up